### PR TITLE
[schema] Update askbot.csv

### PR DESCRIPTION
**Pull request by vchrombie**
This PR updates the askbot schema to the latest format.
Reference to #803

### CODE REVIEW

**Comment by vchrombie** [comment]
Hi @valeriocos 
As of now, I have updated the fields and the `aggregatable` column.

I will squash all the commits once I am done with the PR.
Thanks.

File: schema/askbot.csv [hunk 1]
```diff
@@ -1,48 +1,52 @@
...
-question_answer_count,long
-question_answer_ids,long
-question_has_accepted_answer,long
-question_last_activity_at,date
-question_last_activity_by_id,long
-question_last_activity_by_username,keyword
-question_tags,keyword
-question_title,keyword
-question_view_count,long
-repository_labels,keyword
```
Comment (valeriocos) [inline schema/askbot.csv]: please keep it. This appears when labeling a repo (more info at: https://github.com/chaoss/grimoirelab-sirmordred#projectsjson-, look for `--labels`)

**Review by valeriocos — COMMENTED** [review]
Thank you @vchrombie, I left just a minor comment. Would be possible to add the description of each field (something similar to https://github.com/chaoss/grimoirelab-elk/blob/master/schema/github2_issues.csv)?

Please note that:
- some descriptions can be taken from existing CSVs
- if you think that's too much work we can merge this PR and work on another PR to add the descriptions.

Thanks

File: schema/askbot.csv (hunk 1 above)
Comment (vchrombie) [reply schema/askbot.csv]: Sure, I will update soon. 
Thanks.

**Comment by vchrombie** [comment]
Hi @valeriocos 

> Would be possible to add the description of each field (something similar to https://github.com/chaoss/grimoirelab-elk/blob/master/schema/github2_issues.csv)?
> * some descriptions can be taken from existing CSVs

Yes, I will do that. I was waiting for your approval in the procedure. Also, thanks for the suggestion. I will update you on the PR.

> * if you think that's too much work we can merge this PR and work on another PR to add the descriptions.

Not really, I can push commits to this PR. Once everything is in good shape, I can squash and later we can merge.

**Comment by valeriocos** [comment]
Thanks @vchrombie ! Ping me when the PR is ready!

File: schema/askbot.csv (hunk 1 above)
Comment (vchrombie) [reply schema/askbot.csv]: Resolved now.

Comment (vchrombie) [reply schema/askbot.csv]: Updated.

**Comment by vchrombie** [comment]
Hi @valeriocos 
The `askbot` schema is completed. It is ready for review.
It has 3 commits as of now.
1. https://github.com/chaoss/grimoirelab-elk/pull/812/commits/41f05c9ef3a418bb748a0066e2e067c4ce217299, which updates all the fields with the latest index
2. https://github.com/chaoss/grimoirelab-elk/pull/812/commits/17d2ec61c6f7a0e33aac2bb76961cc93d2d7f8ca, which updates the `aggregatable` column of the fields
3. https://github.com/chaoss/grimoirelab-elk/pull/812/commits/8af354948201faea9647e9207bab350942b8bdd6, which updates the `description` column of the field (this might need a review :sweat_smile: )

Please review it when you are free. After the PR is approved, I will squash all these commits into 1.
Thanks. :slightly_smiling_face:

File: schema/askbot.csv [hunk 2]
```diff
@@ -1,48 +1,53 @@
...
+author_uuid,keyword,true,"Author UUID from SortingHat."
+comment_count,long,true,"Number of comments."
+first_answer,long,true,"Field containing '1' if item is the first answer to question."
+grimoire_creation_date,date,true,"Date when the item was created."
+id,keyword,true,"Id of the item."
+is_accepted_answer,long,true,"Check if item is accepted answer."
+is_askbot_answer,long,true,"Used to separate answers from questions and comments."
+is_askbot_comment,long,true,"Used to separate comments from answers and questions."
+is_askbot_question,long,true,"Used to separate questions from answers and comments."
+metadata__enriched_on,date,true,"Date when the data were enriched."
```
Comment (valeriocos) [inline schema/askbot.csv]: Small typo: data were / data was

**Review by valeriocos — APPROVED** [review]
LGTM, thanks @vchrombie . I left a really minor comment.

Please squash the commits in one and ping me when the PR is ready to be merged. Thanks!

**Comment by vchrombie** [comment]
Hi @valeriocos 
Thanks for the suggestion, updated the OR.

I have squashed the commits too. Please let me know if there are any more changes to do. :slightly_smiling_face:

**Comment by valeriocos** [comment]
Thank you @vchrombie !

**Pull request merged**